source/shared/cpp/ObjectModel/HostConfig.h

```diff
@@ -12,6 +12,7 @@ struct SpacingDefinition
     unsigned int right = 0;
     unsigned int top = 0;
     unsigned int bottom = 0;
+    SpacingDefinition(int left = 0, int right = 0, int top = 0, int bottom = 0):left(left), right(right), top(top), bottom(bottom){};
```

Comment:
awesome, thanks

---
In reply to: [122545652](https://github.com/Microsoft/AdaptiveCards/pull/423#discussion_r122545652) [](ancestors = 122545652,122286906,122282237)